charts/rstudio-connect/ci/complex-values.yaml

```diff
@@ -45,9 +48,18 @@ pod:
       command: ["sleep"]
       args: ["infinity"]
       imagePullPolicy: "IfNotPresent"
+    # This will spin up the chronicle-agent sidecar container
+    - name: chronicle-agent
+      image: ghcr.io/rstudio/chronicle-agent:2023.10.4
+      env:
+      - name: CHRONICLE_SERVER_ADDRESS
+        value: "http://chronicle-server.default.svc.cluster.local"
+      - name: CONNECT_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: connect
+            key: apikey
```

Comment:
This is me adding the chronicle-agent into our rstudio-connect helm values used within ci.
This is a good test addition in it's own right, for using the `sidecar` feature of our helm chart